pkg/internal/ebpf/instrumenter.go

```diff
@@ -431,6 +431,10 @@ func getCgroupPath() (string, error) {
 
 	enabled, err := v2.Enabled()
 	if !enabled {
+		if _, pathErr := os.Stat(filepath.Join(cgroupPath, "unified")); pathErr != nil && os.IsNotExist(pathErr) {
```

Comment:
Actually, I am wondering if we should return in case of any error?

Reply:
What I mean, does it make sense to check if it's not found error? If we couldn't stat the file it will likely mean some sort of permission issue, in which case it might make sense to bail?